pipelines/_ingredients/LSSTCam/DRP.yaml

```diff
@@ -108,6 +109,11 @@ tasks:
       python: |
         from lsst.analysis.tools.atools import *
   # These don't actually work now:
+  photometricMatchVisit:
+    class: lsst.analysis.tools.tasks.photometricCatalogMatch.PhotometricCatalogMatchVisitTask
+    config:
+      extraColumns: ["x", "y", "ap09Flux", "ap09FluxErr"]
+      referenceCatalogLoader.doApplyColorTerms: false
```

Comment:
Can you make this change in `obs_lsst` like you do for `photometricCatalogMatchTask`?